health/health_config.c

```diff
@@ -95,6 +102,11 @@ static inline int rrdcalctemplate_add_template_from_config(RRDHOST *host, RRDCAL
         return 0;
     }
 
+    if ( (!rt->module_match && rt->plugin_match) || (rt->module_match && !rt->plugin_match)) {
+        error("Health configuration for template '%s' has neither \"module\" or \"plugin\", but it is necessary to define both. Ignoring it.", rt->name);
+        return 0;
+    }
```

Comment:
i suggest to remove the check, it could make no sense but it is not prohibited, it is valid config